workflow/scripts/aggregate_counts.py

```
@@ -47,5 +47,14 @@
 # Remove duplicate rows (observed for some libraries)
 df = df.drop_duplicates()
 
+# Make sure that the order of the sample columns is the same as
+# in the design matrix for MAGeCK mle (if used)
+if snakemake.config["stats"]["mageck"]["command"] == "mle":
+    # Read design matrix
+    file = snakemake.config["stats"]["mageck"]["design_matrix"]
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)